src/pages/trainer/training/edit/components/editor.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import {ToolbarContainerComponent} from './toolbar';
+const classNames: any = require('./editorStyles.scss');
+
+interface IProps {
+  content: string;
+  cursorStartPosition: number;
+  shouldSetEditorFocus: boolean;
+  className: string;
```

Comment:
This className is a bit confusing, what is this className for? Could we use a more precise name?